packages/backend/src/models/DashboardModel/DashboardModel.ts

```diff
@@ -394,11 +413,17 @@ export class DashboardModel {
                 `${PinnedListTableName}.pinned_list_uuid`,
                 `${PinnedDashboardTableName}.pinned_list_uuid`,
             )
+            .leftJoin(
+                ValidationTableName,
+                `${ValidationTableName}.dashboard_uuid`,
+                `${DashboardsTableName}.dashboard_uuid`,
+            )
             .select<
-                (GetDashboardQuery & {
-                    space_uuid: string;
-                    spaceName: string;
-                })[]
+                (GetDashboardQuery &
+                    ValidationSummaryQuery & {
+                        space_uuid: string;
```

Comment:
wait, why are we mixing cases here ?

Reply:
this is the type of ValidationSummaryQuery. Should I rename it to something clearer? 
![image](https://github.com/lightdash/lightdash/assets/7611706/361ec7ef-28e5-4dd0-b41f-40cf8cf03748)

```ts
export type ValidationSummaryQuery = {
    validation_error: DbValidationTable['error'];
    validation_created_at: DbValidationTable['created_at'];
};
```

Reply:
😱 didn't even notice, good spot, will also fix that!